README.md

```diff
@@ -1,43 +1,155 @@
 # Cubscout
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/cubscout`. To experiment with that code, run `bin/console` for an interactive prompt.
+Unofficial Ruby client for the [Helpscout API V2](https://developer.helpscout.com/mailbox-api/). 
 
-TODO: Delete this and the text above, and describe your gem
+Cubscout is heavily inspired by [girlscout](https://github.com/omise/girlscout), another unofficial Ruby Client for Helpscout API V1.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add `gem 'cubscout'` to your application's Gemfile, and then run `bundle` to install.
+
+Or install it yourself as `gem install cubscout`.
```

Comment:
Then we should already push it to rubygems right?

Reply:
Well, it already says so in the current readme and it's not true. I wouldn't rush though 🙈

Reply:
Ah, indeed. I agree.